railties/lib/rails/generators/rails/app/templates/config/storage_services.yml

```diff
@@ -0,0 +1,27 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+
+# Use rails secrets:edit to set the AWS secrets (as shared:aws:access_key_id|secret_access_key)
+# amazon:
+#   service: S3
+#   access_key_id: <%= Rails.application.secrets.dig(:aws, :access_key_id) %>
+#   secret_access_key: <%= Rails.application.secrets.dig(:aws, :secret_access_key) %>
+#   region: us-east-1
+#   bucket: your_own_bucket
+
+# Remember not to checkin your GCS keyfile to a repository
+# google:
+#   service: GCS
+#   project: your_project
+#   keyfile: <%= Rails.root.join("path/to/gcs.keyfile") %>
+#   bucket: your_own_bucket
+
+# mirror:
+#   service: Mirror
+#   primary: local
+#   secondaries: [ amazon, google ]
```

Comment:
Should this template be renamed to match the name of the file it’s used to generate?